simple-blog/src/app/blog-navbar/blog-navbar.component.sass

```diff
@@ -3,8 +3,26 @@
   display: flex
   justify-content: space-between
   #nav-title
-    // ToDO: will need to be restyled in the future
-    width: 200px
+    // Home Page Button Div
+    border: 3px solid black
+    border-radius: 10px
+    width: 150px
+    display: flex
+    flex-direction: column
+    align-items: center
+    background-color: #d4cb20
+    #heading-link
+      // Home Page Button
+      text-decoration: none
+      border-radius: 10px
+      width: 100%
+      height: 100%
+      background: transparent
+      font: inherit
+      &:hover
+        background-color: #d6cf42
```

Comment:
Could you make this color a css variable in variables.sass. Maybe name it something like "--color-primary-hover". Or whatever you think makes sense/follows the established pattern.